webapp/src/modules/translation/locales/es.json

```
@@ -1306,6 +1306,10 @@
     "campaign_browser": {
       "title": "Esenciales del Music Festival",
       "subtitle": "Desde Emotes de surf en la multitud hasta wearables brillantes, encuentre todos los artículos del festival que necesita para #DCLMF23 aquí"
+    },
+    "empty_campaign": {
+      "title": "Los creadores aún no han agregado sus artículos aquí.Mientras tanto, consulte las creaciones de la comunidad en la pestaña 'Collectibles' para encontrar algo que le guste.",
```

If I'm not wrong, we're using `Coleccionables` for the Collectibles word:
```suggestion
      "title": "Los creadores aún no han agregado sus artículos aquí. Mientras tanto, consulte las creaciones de la comunidad en la pestaña 'Coleccionables' para encontrar algo que le guste.",
```